torchtune/datasets/__init__.py

```diff
@@ -8,6 +8,7 @@
 from torchtune.datasets._chat import chat_dataset, ChatDataset
 from torchtune.datasets._cnn_dailymail import cnn_dailymail_articles_dataset
 from torchtune.datasets._concat import ConcatDataset
+from torchtune.datasets._finetune import SFTDataset
```

Comment:
Sorry to be that guy but imo it's a bit confusing that our canonical dataset does not match the pattern of all our other datasets (class name is the capitalized version of the filename). I'm good with the name `SFTDataset`, but then maybe we should rename the file too?

Reply:
yeah I missed that. will include that change in the subsequent PRs